packaging/android/android-build-wrapper.sh

```diff
@@ -19,8 +19,8 @@ USE_X=$(case $- in *x*) echo "-x" ;; esac)
 
 # these are the current versions for Qt, Android SDK & NDK:
 
-QT_VERSION=5.9.1
-LATEST_QT=5.9.1
+QT_VERSION=5.9
+LATEST_QT=5.9.2
```

Comment:
That's a really bad idea. 5.9.2 is broken with Kirigami and immediately crashes at start.

Reply:
Didn't know that. I generally build the desktop ui locally when I test things on android, and 5.9.2 was the Qt version I had laying around. I'll skip that then.

Reply:
@glance-: QTBUG-64017